src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-play-video-overlay/main.js

```diff
@@ -0,0 +1,224 @@
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+/**
+ * @class MediaPlayVideoOverlay
+ * Class which shows overlays for playing videos
+ * @constructor
+ */
+define([
+    'services/sulumedia/media-manager'
+], function(MediaManager) {
+
+    'use strict';
+
+    var namespace = 'sulu.media-play-video.',
+
+        defaults = {
+            instanceName: ''
+        },
+
+        constants = {
+            loadingClass: 'loading',
+            loaderClass: 'media-play-video-loader',
+            playVideoId: 'media-play-video-id'
+        },
+
+        templates = {
+            video: function(videoUrl) {
+                return [
+                    '<div class="media-play-video-container">',
+                    '   <video id="', constants.playVideoId, '" controls>',
+                    '       <source src="', videoUrl, '" />',
+                    '   </video>',
+                    '</div>'
+                ].join('');
+            }
+        },
+
+        /**
+         * raised when the overlay get closed
+         * @event sulu.media-play-video.closed
+         */
+        CLOSED = function() {
+            return createEventName.call(this, 'closed');
+        },
+
+        /**
+         * raised when component is initialized
+         * @event sulu.media-play-video.closed
+         */
+        INITIALIZED = function() {
+            return createEventName.call(this, 'initialized');
+        },
+
+        /** returns normalized event names */
+        createEventName = function(postFix) {
+            return namespace + (this.options.instanceName ? this.options.instanceName + '.' : '') + postFix;
+        };
```

Comment:
Instead of creating the events like this you could also use the quite new event system: http://docs.sulu.io/en/latest/bundles/admin/javascript-hooks/events.html